src/lib/hooks/useZnsDomain.tsx

```diff
@@ -108,7 +111,7 @@ export const useZnsDomain = (domainId: string): UseZnsDomainReturn => {
 
 			setLoading(false);
 		} catch (e) {}
-	}, [domainId]);
+	}, [domainId, chainId]);
```

Comment:
Could be wrong, but you may be able to swap `chainId` for `sdk`, since `sdk` will be changing on chain ID. This would remove the need for a `chainId` parameter on the hook.

Reply:
For some reason the SDK instance looks same. I'll take a recheck once